vaadin-board.html

```diff
@@ -75,8 +75,9 @@
       /**
        * @namespace Vaadin
        */
+      window.Vaadin = window.Vaadin || {};
       window.Vaadin.BoardElement = BoardElement;
-      if (window.Vaadin.runIfDevelopmentMode) {
+      if (window.Vaadin && window.Vaadin.runIfDevelopmentMode) {
```

Comment:
It's unnecessary the `window.Vaadin` check, since it's always defined in line 78